docs/source/upgrade-guide.md

```diff
@@ -12,6 +12,30 @@ myst:
 This upgrade guide lists all breaking changes in `plone.restapi`.
 It explains the steps that are needed to upgrade to the latest version.
 
+## Upgrading to `plone.restapi` 9.x
```

Comment:
@davisagli can you check that what I wrote here is correct?

Reply:
@tisto yes, basically, but one nuance: The endpoint accepts multiple UIDs. Before, objects with no breaches were left out of the result. Now they are always included, so that we can also show how many items are contained. So, the old result might not be an empty list of multiple UIDs were requested (I.e. multiple checkboxes selected to delete in folder contents) and some of them have breaches

Reply:
@davisagli not sure I fully get this. Could you add this to the upgrade guide?

Reply:
@davisagli I merged this PR, feel free to add to main or create a new PR.